src/ray/raylet/node_manager.cc

```diff
@@ -735,6 +739,83 @@ void NodeManager::DispatchTasks(
   local_queues_.MoveTasks(assigned_task_ids, TaskState::READY, TaskState::RUNNING);
 }
 
+std::pair<std::shared_ptr<Worker>, bool> NodeManager::GetWorker(
+    const WorkerID &worker_id) {
+  bool is_worker = true;
```

Comment:
instead of returning a pair here, I think it's clearer to put `is_worker` in `Worker` class.